Packages/com.llealloo.audiolink/Runtime/Scripts/AudioReactiveBlendshapes.cs

```diff
@@ -0,0 +1,47 @@
+﻿using UnityEngine;
+
+namespace AudioLink
+{
+#if UDONSHARP
+    using UdonSharp;
+    using UnityEngine.Rendering.PostProcessing;
+
+    public class AudioReactiveBlendshapes : UdonSharpBehaviour
+#else
+    public class AudioReactiveBlendshapes : MonoBehaviour
```

Comment:
Use [RequireComponent(typeof(SkinnedMeshRenderer))] ?

Reply:
SkinnedMeshRenderer is now required